rca/project_styleguide/templates/patterns/organisms/stat-block/stat-block.html

```diff
@@ -1,36 +1,41 @@
-<section class="section section--stat-block bg bg--stat-block" data-stat-block style="--stat-block-bg-image: url('{{bg_image_url}}')">
+{% load wagtailimages_tags %}
+{% image stats_block.background_image max-1000x500 as bg_image %}
+<section class="section section--stat-block bg bg--stat-block" data-stat-block style="--stat-block-bg-image: url('{{bg_image.url}}')">
     <div class="section__notch section__notch--transparent section__notch--opposite">
         <div class="section__notch-fill section__notch-fill--second-col"></div>
     </div>
     <div class="section__container stat-block">
         <div class="stat-block__header grid">
-            <h3 class="stat-block__heading heading heading--three">{{heading}}</h3>
+            <h3 class="stat-block__heading heading heading--three">{{stats_block.title}}</h3>
```

Comment:
Why do we suddenly format variables without spaces around their names? Can we keep spaces since that's more Django-like please?